tools/e2etesting/steps/deployplatform.yml

```diff
@@ -17,6 +17,45 @@ steps:
     artifact: 'iai'
     path: $(BasePath)
 
+- task: AzureKeyVault@1
```

Comment:
Let's keep in mind that we need to be able to consume variables from pipeline UI as well and this particular step overrides those variable in all cases, even when there were values provided.